### DISCUSSION TITLE
Need recommendations on a POS system for integration

### DISCUSSION THREAD

**Discussion by mfreeman451** [inventree/InvenTree#7567] [General]
Anyone have any recommendations for a POS system that they've had experience integrating InventTree successfully with? I can code and am willing to create my own integrations, but this is definitely unfamiliar territory.

**Comment by jacobburrell** [comment]
Also looking to see if Inventree could work with restaurant Inventory management. 

Esp. with perishables like food. Seems like something that is quite specific and ideally integrated into a POS.

**Reply by SchrodingersGat** [reply]
@jacobburrell not sure about that, if you can layout any specific requirements or features you are looking for?
